src/main/java/org/terasology/launcher/settings/LauncherSettingsValidator.java

```diff
@@ -29,19 +29,37 @@
 public final class LauncherSettingsValidator {
     private static final Logger logger = LoggerFactory.getLogger(LauncherSettingsValidator.class);
 
+    private static String removeUnsupportedJvmParameters(final String currentParams) {
+        String params = currentParams;
+
+        final String[] deprecatedParams = {"-XX:\\+UseParNewGC", "-XX:\\+UseConcMarkSweepGC", "-XX:ParallelGCThreads=10"};
+
+        for (String deprecatedParam : deprecatedParams) {
+            params = params.replaceAll(deprecatedParam, "");
```

Comment:
I suggest `removeAll(deprecatedParam)` here, rather than replacing with empty strings.

Reply:
That's the Java way of `removeAll` =/

Reply:
this thing? https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/List.html#removeAll(java.util.Collection)